frontend/src/components/TheHeader/TheHeader.vue

```diff
@@ -0,0 +1,98 @@
+<template>
+  <header class="theHeader">
+    <div class="theHeader__nav-wrapper">
+      <div class="theHeader__logo-wrapper">
+        <router-link to="/">
+          <t-icon class="theHeader__icon" icon="header-logo" />
+        </router-link>
+      </div>
+      <nav class="theHeader__nav">
+        <router-link class="theHeader__dashboard-wrapper" to="/dashboard">
+          <t-button
+            class="theHeader__nav-name"
+            icon="home"
+            iconPosition="left"
+            type="subtle"
+            >Dashboard</t-button
+          >
+        </router-link>
+      </nav>
+    </div>
+    <t-dropdown
+      title="Ongoing Tasks"
+      isLoading
+      isHeaderDropdown
+      :tasksList="tasksList"
+    />
+  </header>
+</template>
+
+<script>
+import TButton from "../common/Button/TButton.vue";
+import TDropdown from "../common/Dropdown/TDropdown.vue";
+import TIcon from "../common/Icon/TIcon.vue";
+
+export default {
+  components: { TIcon, TDropdown, TButton },
+  data() {
+    return {
+      tasksList: [
+        {
+          title: "Disconnected talos-prod-US-bbkgi (5 nodes)",
+          time: "06:12:22",
+          description:
+            "kubernetes.namespace.name=’istio-system’and kubkubernetes.namespace.name=’istio-system’and kubkubernetes.namespace.name=’istio-system’and kub",
+        },
+        {
+          title: "Disconnected talos-prod-US-bbkgi (5 nodes)",
+          time: "06:12:22",
+          description:
+            "kubernetes.namespace.name=’istio-system’and kubkubernetes.namespace.name=’istio-system’and kubkubernetes.namespace.name=’istio-system’and kub",
+        },
+      ],
+    };
```

Comment:
I we're using vue 3 this data should be returned from `setup()` method.

And I don't think we should use mock data here.
I guess ongoing tasks list should be kept as a global reactive list somewhere on the frontend.

I would suggest to use real Kubernetes upgrade tasks here instead of the mocks.